src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="star stars--0">
+      <div class="stars__star non-active"></div>
+      <div class="stars__star non-active"></div>
+      <div class="stars__star non-active"></div>
+      <div class="stars__star non-active"></div>
+      <div class="stars__star non-active"></div>
+    </div>
+    <div class="star stars--1">
+      <div class="stars__star active"></div>
+      <div class="stars__star non-active"></div>
+      <div class="stars__star non-active"></div>
+      <div class="stars__star non-active"></div>
+      <div class="stars__star non-active"></div>
```

Comment:
All star elements should have the same class in HTML.
You should change the background image in CSS based on parent modifier (`stars--1`, `stars--2` etc)